v3/position_test.go

```diff
@@ -1,17 +1,27 @@
 package geohex
 
 import (
+	"fmt"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("Position", func() {
 
 	It("should generate centroids", func() {
-		pos := &Position{4, -5, zooms[0]}
+		pos := &Position{X: 4, Y: -5, z: zooms[0]}
 		cnt := pos.Centroid()
 		Expect(cnt.E).To(BeNumerically("~", 20037508.3, 0.1))
 		Expect(cnt.N).To(BeNumerically("~", -1285406.8, 0.1))
 	})
 
+	for _, tc := range loadPosition2HexTestCases() {
```

Comment:
I don't think it's going to be clearer. I'd have to transform the array of testcases into the array of `Entry()`'s (unless  I return those from the `load...` functions) and then pass it to `DescribeTable`, so it would be the same thing but with more code.
That documentation says

> While it’s easy to roll your own table driven tests using simple data structures and a for loop, this layer of DSL makes it particularly easy to write and manage table driven tests.

And I'd stuck to that "while it's easy" part with the KISS principle :)